cmd/cloud/cluster_flag.go

```diff
@@ -240,6 +245,7 @@ func (flags *clusterResizeFlags) addFlags(command *cobra.Command) {
 	command.Flags().StringVar(&flags.nodeInstanceType, "size-node-instance-type", "", "The instance type describing the k8s worker nodes. Overwrites value from 'size'.")
 	command.Flags().Int64Var(&flags.nodeMinCount, "size-node-min-count", 0, "The minimum number of k8s worker nodes. Overwrites value from 'size'.")
 	command.Flags().Int64Var(&flags.nodeMaxCount, "size-node-max-count", 0, "The maximum number of k8s worker nodes. Overwrites value from 'size'.")
+	command.Flags().StringSliceVar(&flags.nodeGroups, "node-groups", nil, "The nodegroups to resize. If not provided, all nodegroups will be resized.")
```

Comment:
Hmm, this may be a bit dangerous for default behavior. I think it may be better to force names to prevent a situation like accidentally resizing all prod cluster nodes.

Reply:
Right. Thanks for pointing out